src/services/bundlers/bundlerSwitcher.ts

```diff
@@ -42,7 +43,10 @@ export class BundlerSwitcher {
     return this.noStateUpdateStatuses.includes(this.getSignAccountOpStatus())
   }
 
-  canSwitch(bundlerError: Error | null): boolean {
+  canSwitch(acc: Account, bundlerError: Error | null): boolean {
+    // no fallbacks for EOAs
+    if (!acc.creation) return false
```

Comment:
Maybe `isBasicAccount(acc, state)`?

Reply:
Haven't started the bundler switchers, yet